impl/src/test/java/com/sun/faces/mock/MockServletContext.java

```diff
@@ -39,111 +40,116 @@
 // Mock Object for ServletContext (Version 4.0)
 public class MockServletContext implements ServletContext {
 
-    private Hashtable attributes = new Hashtable();
-    private Hashtable parameters = new Hashtable();
+    private Hashtable<String, Object> attributes = new Hashtable<>();
+    private Hashtable<String, String> parameters = new Hashtable<>();
 
     // --------------------------------------------------------- Public Methods
     public void addInitParameter(String name, String value) {
         parameters.put(name, value);
     }
 
     // ------------------------------------------------- ServletContext Methods
+    @Override
     public Object getAttribute(String name) {
         return (attributes.get(name));
     }
 
-    public Enumeration getAttributeNames() {
+    @Override
+    public Enumeration<String> getAttributeNames() {
         return (attributes.keys());
     }
 
+    @Override
     public ServletContext getContext(String uripath) {
         throw new UnsupportedOperationException();
     }
 
+    @Override
     public String getContextPath() {
         return "test";
     }
 
+    @Override
     public String getInitParameter(String name) {
-        return ((String) parameters.get(name));
+        return (parameters.get(name));
```

Comment:
Unnecessary parenthesis should also be removed over all place.

Reply:
Sure. My bad.